src/services/Services.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { servicesConstnat } from '../constant/AppConstant'
+
+const Services = () => {
+
+    const serviceElements = servicesConstnat?.map(service => {
```

Comment:
You have not applied the correct background color to the entire services section. Confirm with the design and put the correct color.